com.wireframesketcher.WireframeSketcher.json

```diff
@@ -0,0 +1,61 @@
+{
+  "id": "com.wireframesketcher.WireframeSketcher",
+  "runtime": "org.gnome.Platform",
+  "runtime-version": "46",
+  "sdk": "org.gnome.Sdk",
+  "command": "WireframeSketcher",
+  "separate-locales": false,
+  "tags": [
+    "proprietary"
+  ],
+  "rename-appdata-file": "wireframesketcherstudio.appdata.xml",
+  "rename-desktop-file": "wireframesketcherstudio.desktop",
+  "rename-icon": "WireframeSketcher",
+  "finish-args": [
+    "--filesystem=home",
+    "--filesystem=/tmp:rw",
```

Comment:
why is `/tmp` on host needed ? the sandbox has its own tmp directory.

Reply:
Removed the /tmp filesystem requirement.